### ISSUE TITLE
[Feature Request]: Add option for predicting individual growth class ID

### ISSUE THREAD

**Issue by salupere** [jasp-stats/jasp-issues#2762]
### Description

It would be handy to have and option to predict and save (as a separate variable to the dataset) growth trajectory class for each case separately. Lavaan provides it through lavaan::lavPredict function and lcmm library provides it through model object extraction (mymodel[["pprob"]]$class) 

### Is your feature request related to a JASP module?

SEM

### Describe the solution you would like

A checkbox in main dialog, under "Additional output" tab, there is a checkbox "Add predicted growth trajectory class"

**Comment by tomtomme** [comment]
@salupere 
thx for the request. This is for the latent growth submodule of SEM?

@juliuspfadt 
The feature request seems valid, but I am not sure. Can you confirm?

**Comment by salupere** [comment]
yes, it would be nice option to have it in the latent growth submodule of sem

**Comment by juliuspfadt** [comment]
If it works out just as @salupere suggests, then it will be relatively straightforward to implement.
